src/ResourceManager/MachineLearning/AzureRM.MachineLearning.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.11.1'
```

Comment:
@NonStatic2014 we will change this version number according to the release, so you can revert this change.